docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -375,6 +375,9 @@ include::{generated-dir}/config/quarkus-vertx-http-config-group-access-log-confi
 |Vert.x MDC data (e.g. 'traceId' for OpenTelemetry)                           |          | `%{X,mdc-key}`
 |===
 
+=== Exclude Paths from HTTP Access Log
+Use `quarkus.http.access-log.exclude-pattern=/some/path/.*` in your `application.properties` to exclude all entries concerning the path `/some/path/...` (_including subsequent paths_) from the log. Since a regex is being used here, you can use tools like https://regex101.com[regex101] to prototype or test your patterns.
+
```

Comment:
I think it would be better if this were a `TIP`, i.e.:

```
[TIP]
====
....
====
```

Reply:
Ok, I have changed the formatting and created a [new PR](https://github.com/quarkusio/quarkus/pull/31128).